geomstats/geometry/_hyperbolic.py

```diff
@@ -455,21 +447,16 @@ class HyperbolicMetric(RiemannianMetric):
         Dimension of the hyperbolic space.
     default_coords_type : str, {'extrinsic', 'intrinsic', etc}, optional
         Default coordinates to represent points in hyperbolic space.
-    scale : int, optional
-        Scale of the hyperbolic space, defined as the set of points
-        in Minkowski space whose squared norm is equal to -scale.
     """
 
-    def __init__(self, dim, scale=1, default_coords_type="extrinsic"):
+    def __init__(self, dim, default_coords_type="extrinsic"):
```

Comment:
I think now we can simplify this class my simply removing everything (maybe we just keep it due to mathematical reasoning, i.e. the children are hyperbolic metrics).

In children (at least in `Hyperboloid`) we need then to update `_inner_product` and `_squared_norm` to be simply `inner_product` and `squared_norm`.

Reply:
Yes, I agree. Done.